### PR TITLE
Add `en-sync-check.sh` for daily-check

### PR DESCRIPTION
**Pull request by jay-kim-kor**
# Description

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

## Summary by CodeRabbit

- **New Features**
	- Introduced a synchronization check tool for Klaytn nodes, enabling users to verify the sync status between local and public nodes.

### CODE REVIEW

**Comment by martinkersner** [comment]
@jay-kim-kor I think it would be worth to check the comments from @coderabbitai. Do you think they are irrelevant?

**Comment by jay-kim-kor** [comment]
> @jay-kim-kor I think it would be worth to check the comments from @coderabbitai. Do you think they are irrelevant?

I've checked that comments but, I didn't apply they recommendation.
I think it's not irrelevant. But I just don't agree to it's need double quote in `echo`.

File: monitor/scripts/en-sync-check.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	# Get log from JSON-RPC node
+	response=$(gcloud compute ssh orakl-cypress-prod-node \
+		--project=orakl-cypress-prod \
+		--tunnel-through-iap \
+		-- -t tail -1 /data/pruning/log/kend.out)
+
+	# Get block number from log
+	# If col 8 string starts with "number=",
+	# then print the value only integer part
+	$our_klay_blockNumber = $(echo $response | awk '$8 ~ /^number*/ { sub(/^number=/, "", $8); print $8 }')
```
Comment (martinkersner) [inline monitor/scripts/en-sync-check.sh:46]: @jay-kim-kor Is the `$` in front of `our_klay_blockNumber` on purpose? Also, I think that the spaces around `=` should be removed. https://unix.stackexchange.com/questions/258727/spaces-in-variable-assignments-in-shell-scripts

Comment (jay-kim-kor) [reply monitor/scripts/en-sync-check.sh]: No that's not on purpose. Thank you!

**Pull request merged**